app/assets/javascripts/admin/addon/controllers/admin-area-settings-base.js

```diff
@@ -0,0 +1,13 @@
+import { tracked } from "@glimmer/tracking";
+import Controller from "@ember/controller";
+import { action } from "@ember/object";
+
+export default class AdminAreaSettingsBaseController extends Controller {
+  @tracked filter = "";
+  queryParams = ["filter"];
+
+  @action
+  filterChangedCallback(filter) {
```

Comment:
Sorry I know this is annoying and I should have raised it earlier, but `adminSettingsFilterChangedCallback` might be better for "search juice" for people to find this base controller.